### PR TITLE
build(bazel): update to rules_typescript 0.16.0 & update to tagged rules_webtesting 0.2.1

### PR DESCRIPTION
**Pull request by gregmagolan**
Also check that rules_typescript version is at least 0.16.0

### CODE REVIEW

File: integration/bazel/WORKSPACE [hunk 1]
```diff
@@ -81,8 +81,9 @@ browser_repositories(
     firefox = True,
 )
 
-load("@build_bazel_rules_typescript//:defs.bzl", "ts_setup_workspace")
+load("@build_bazel_rules_typescript//:defs.bzl", "ts_setup_workspace", "check_rules_typescript_version")
 
+check_rules_typescript_version("0.16.0")
```
Comment (alexeagle) [inline integration/bazel/WORKSPACE]: what's the point of checking the version if we've used `http_archive` above and already know this is the version?

Comment (gregmagolan) [reply integration/bazel/WORKSPACE]: Oh.. right. I meant to put that check into ng_setup_workspace for downstream. Thanks

File: WORKSPACE [hunk 2]
```diff
@@ -20,16 +20,16 @@ http_archive(
...
 )
 
 http_archive(
     name = "build_bazel_rules_typescript",
-    url = "https://github.com/bazelbuild/rules_typescript/archive/1d9a4b0087f307e31af91e2b221a6447288994c6.zip",
-    strip_prefix = "rules_typescript-1d9a4b0087f307e31af91e2b221a6447288994c6",
-    sha256 = "e17ac3f33d5d3cd2a0c385c4fd28b814d0ad46c6c67ccaef97160be99d7a24eb",
+    url = "https://github.com/bazelbuild/rules_typescript/archive/0.16.0.zip",
+    strip_prefix = "rules_typescript-0.16.0",
+    # sha256 = "e17ac3f33d5d3cd2a0c385c4fd28b814d0ad46c6c67ccaef97160be99d7a24eb",
```
Comment (alexeagle) [inline WORKSPACE]: hmm, put the sha back?

**Review by alexeagle — APPROVED** [review]

File: WORKSPACE (hunk 2 above)
Comment (gregmagolan) [reply WORKSPACE]: 🤦‍♂️  Thanks.. Must be Friday

**Comment by benlesh** [comment]
@alexeagle ... I had issues merging this into both master and patch, we may need a separate PR for the patch branch. cc @IgorMinar

**Pull request closed without merging**